docs/user-guide/ze-usage-tips.md

```diff
@@ -3,27 +3,35 @@
 Make the best use of Zowe Explorer with the following tips.
 
 ## Data sets, USS, and jobs persistence settings
+
 You can store any data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**.
 
 ## Identify syntax errors with a syntax highlighter
+
 Zowe Explorer supports a syntax highlighter for data sets. To enhance the experience of using the extension, you can download an extension that highlights syntax.
```

Comment:
For lean content, change "you can download an extension that highlights syntax"
to 
"download an extension that highlights syntax."